src/pls/fs/git.py

```diff
@@ -89,14 +90,18 @@ def get_git_statuses(git_root: Path) -> dict[Path, str]:
     for line in status_lines:
         status = line[0:2]
 
-        components: list[str] = shlex.split(line[3:])
+        components: list[str] = shlex.split(line[3:], posix=(platform != "win32"))
         if len(components) == 1:
             path_str = components[0]
         elif len(components) == 3:
             _, __, path_str = components
+        # If the return code was 0 but there were no results
+        # then no changes where made
+        elif len(components) == 0 and proc.returncode == 0:
+            path_str = ""
```

Comment:
This should be handled by commit 74b85fe.

Reply:
Ok, I took it out.